lib/ansible/plugins/filter/core.py

```diff
@@ -397,9 +396,9 @@ def comment(text, style='plain', **kw):
         # Prepend each line of the text with the decorator
         text.replace(
             p['newline'], "%s%s" % (p['newline'], p['decoration'])))).replace(
-                # Remove trailing spaces when only decorator is on the line
-                "%s%s" % (p['decoration'], p['newline']),
-                "%s%s" % (p['decoration'].rstrip(), p['newline']))
+        # Remove trailing spaces when only decorator is on the line
```

Comment:
You are right of course, this is my mistake caused by using auto-format IDE function.